build.gradle

```diff
@@ -9,18 +9,25 @@ plugins {
     id 'com.install4j.gradle' version '8.0.7'
 }
 
+configurations.all {
+    resolutionStrategy.cacheChangingModulesFor 0, "seconds"
+}
+
 repositories {
     jcenter()
+    maven {
+        url = uri("https://oss.sonatype.org/content/repositories/snapshots/")
+    }
 }
 
 dependencies {
     implementation 'org.scribe:scribe:1.3.0'
     implementation 'org.hsqldb:hsqldb:2.4.1'
     implementation 'com.google.code.gson:gson:2.8.6'
     implementation 'com.squareup.okhttp3:okhttp:4.3.1'
-    implementation 'com.github.weisj:darklaf-core:2.2.3'
-    implementation 'com.github.weisj:darklaf-theme:2.2.3'
-    implementation 'com.github.weisj:darklaf-property-loader:2.2.3'
+    implementation 'com.github.weisj:darklaf-core:latest.integration'
+    implementation 'com.github.weisj:darklaf-theme:latest.integration'
+    implementation 'com.github.weisj:darklaf-property-loader:latest.integration'
```

Comment:
Are the changes here required for this PR to work?  I'd rather keep dependencies on fixed released versions.

Reply:
They are necessary for now. I am going to release the next version later today wich include the necessary fixes.

Reply:
Apologies, I only saw your note to that end just now – perfect, I am happy to wait for the release, no rush.